mapbox/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     package="org.odk.collect.mapbox">
 
-</manifest>
\ No newline at end of file
+    <application>
+
+        <!-- Disable Mapbox initializers to prevent them running on devices that don't support it -->
+        <provider
+            android:name="androidx.startup.InitializationProvider"
+            android:authorities="${applicationId}.androidx-startup"
+            android:exported="false"
+            tools:node="merge">
+
+            <meta-data
```

Comment:
I couldn't find these initializer classes referenced in the Mapbox docs. This meant I had to debug inside Android library code to work out what initializers were being run. @lognaturel are we able to send some feedback to Mapbox about this?